docs/DeveloperGuide.md

```diff
@@ -325,7 +368,48 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**Use case: Find client**
+
+**MSS**
+
+1.  User requests to find a client
+2.  ConnectCare lists all clients that match the keyword
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. There is no given keyword.
+
+    * 1a1. AddressBook shows an error message.
+
+      Use case ends.
+
+**Use case: Clear all clients**
+
+**MSS**
+
+1.  User requests to clear all clients
+2.  ConnectCare clears all clients
```

Comment:
Should the system confirm with the user first before clearing all clients?

Reply:
Thats probably a good idea